.github/actions/test-khiops-install/action.yml

```diff
@@ -1,74 +1,57 @@
 ---
 name: Test Khiops Installation
-description: Tests a Khiops installation on the Iris dataset
+description: Check the outputs of Khiops in the console
 runs:
   using: composite
+  # Test if the output of Khiops in the console is as expected (no mpi or java errors)
+  # We expect only one line in stdout+stderr when running 'khiops -v'
   steps:
-    # Python install: we don't use the setup-python action because of the following error:
-    # python3: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by python3)
-    - name: Python setup for Linux
-      if: runner.os == 'Linux'
+    - name: Print status for debugging
       shell: bash
+      # We force the process number to 4 because the runner has only 2 proc.
+      # With 2 proc khiops-env switches to serial and mpi is not used.
+      # The env var OMPI_MCA_rmaps_base_oversubscribe is for openmpi it corresponds to
+      # the flag --oversubscribe (Nodes are allowed to be oversubscribed)
+      env:
+        KHIOPS_PROC_NUMBER: '4'
+        OMPI_MCA_rmaps_base_oversubscribe: 'true'
```

Comment:
You can just leave `true` as a value (without quotes).

Reply:
This is weird, as we have these w/o quotes on `khiops-python` and pre-commit passes: e.g. https://github.com/KhiopsML/khiops-python/blob/a1952b218a18696d3c9930867103581820c3e103/.github/workflows/pip.yml#L72.

Reply:
And, locally, on the `khiops` repo as well, `yamlfix` succeeds when these 2 env vars are unquoted.

Reply:
Oh, I see, it's `check-jsonschema` that fails. You need to update its version to 0.28.6. Thus it will succeed with these without quotes (which is valid YAML).

Reply:
done